web_drag_drop_upload/__init__.py

```diff
@@ -0,0 +1 @@
+import controllers
```

Comment:
Hi @agenterp Thanks for your contribution to the OCA. 
Please follow the guidelines from:
https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md
I miss the README file and short headers in your py files following: https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3